### PR TITLE
improve docs/PR template

### PR DESCRIPTION
**Pull request by nkolev92**
## Bug

Fixes: None
- [ ] Regression
Last working release:
How are we preventing it in the future:

## Fix

Details: 
Improving the docs. Readme/contributing were a bit out of date. 
Taking another stab at a PR template. 
Any feedback is welcome.

Also I'm using the PR template I'm proposing for this PR :)

**Update: GH treats this as a "task list" so it says 0/5 if you look at the PR tracker. So I'm ok with not making any checkbox changes, but I'll wait on feedback from everyone on this**

## Testing

- [ ] Tests Added
Reason for not adding tests: Docs
- [ ] Automation
- [ ] Manual validation
- [ ] Vendor run

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Setup
 
-After you clone this repository run build.ps1, you might have to run build.ps1 -CleanCache if the build doesn't work (because we sometimes update temporary packages in place).
+After you clone this repository run build.ps1, you might have to run build.ps1.
```
Comment (zivkan) [inline CONTRIBUTING.md]: run configure.ps1? otherwise using build.ps1 twice with different contexts sounds weird.

File: README.md [hunk 2]
```diff
@@ -6,7 +6,7 @@
 
 This repo contains the following clients:
   * [NuGet CLI](https://docs.nuget.org/ndocs/tools/nuget.exe-cli-reference)
-  * [NuGet Package Manager for Visual Studio 2017](https://docs.nuget.org/ndocs/tools/package-manager-ui)
+  * [NuGet Package Manager for Visual Studio](https://docs.nuget.org/ndocs/tools/package-manager-ui)
```
Comment (zivkan) [inline README.md]: https://docs.microsoft.com/en-us/nuget/tools/package-manager-ui?

**Review by zivkan — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (nkolev92) [reply README.md]: The links are fwed but I can double check them.

Comment (nkolev92) [reply README.md]: All the links go where they are supposed to.

File: CONTRIBUTING.md (hunk 1 above)
Comment (nkolev92) [reply CONTRIBUTING.md]: fixed

**Comment by nkolev92** [comment]
Thanks for taking a look @zivkan 

Any thoughts on the PR template?
Or should I revert to the old one?

File: README.md (hunk 2 above)
Comment (zivkan) [reply README.md]: They are forwarded, but there's a higher risk of the docs.nuget.org link breaking than the current docs URL, unless you think nuget's docs are going to move back to nuget.org at some point in the future.

Comment (nkolev92) [reply README.md]: Good point, I missed that it's docs.**nuget**.

Will fix that.

**Comment by zivkan** [comment]
I didn't notice that GitHub puts a 0/5 progress on the PR list page, but since you pointed it out, I don't like it since it makes the PR look like a WIP, even though it's not.

Also, I was confused why "automation" is a possible reason why tests aren't added, but now I think maybe it's supposed to be a checklist of which types of tests were added? Maybe if you move the 3 test types to a nested list under tests added, and add the reason why tests were not added to the end, it might be less confusing?

**Comment by nkolev92** [comment]
Yeah I was going for a checklist of validations done, in addition to a test added :)

I will change it back then, with some minor language changes.

**Pull request merged**